RadarSDK/RadarAPIClient.h

```diff
@@ -65,27 +64,32 @@ typedef void (^_Nonnull RadarSyncLogsAPICompletionHandler)(RadarStatus status);
 
 + (NSDictionary *)headersWithPublishableKey:(NSString *)publishableKey;
 
-+ (RadarMeta *_Nullable)parseMeta:(NSDictionary *_Nullable)res;
```

Comment:
I like the rename from meta 👍

Reply:
Yeah, tried to make this consistent across iOS and Android